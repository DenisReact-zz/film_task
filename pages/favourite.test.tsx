import Favourite from "./favourites";
import { screen, render } from "@testing-library/react";
import "isomorphic-fetch";

describe("Tests for favourite page", () => {
  it("check layout favourite page", () => {
    render(<Favourite />);
    const favourite = screen.getByTestId("favourite");
    expect(favourite).toMatchSnapshot();
  });
});
