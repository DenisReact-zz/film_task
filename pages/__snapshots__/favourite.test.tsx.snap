// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for favourite page check layout favourite page 1`] = `
<div
  class="container"
  data-testid="favourite"
>
  <p
    class="home"
  >
    âˆŸ
  </p>
  <h2>
    Your Favourite
  </h2>
  <div
    class="blocks"
  >
    It's empty
  </div>
</div>
`;
